winrm.go

```diff
@@ -27,6 +27,7 @@ const (
 	NTLM
 	Kerberos
 )
+const WINRM_HTTP_PORT = 5985
```

Comment:
It's a detail but this is unused... so I'm wondering why you're adding it.

Reply:
addressed